composer.json

```diff
@@ -6,10 +6,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "ec-europa/oe_core": "*"
+        "ec-europa/oe_core": "*",
```

Comment:
We should remove the dependency from oe_core, right?

Reply:
Yes that's correct, we should remove this.